content/blog/2015-12-18-react-components-elements-and-instances.md

```diff
@@ -294,15 +294,15 @@ class Button extends React.Component {
 }
 ```
 
-When a component is defined as a class, it is a little bit more powerful than a function component. It can store some local state and perform custom logic when the corresponding DOM node is created or destroyed.
+Lorsqu'un composant est défini en tant que classe, il est un peu plus puissant qu'un composant de fonction. Il peut stocker un état local et exécuter une logique personnalisée lorsque le nœud DOM correspondant est créé ou détruit.
```

Comment:
"un composant de fonction" ça me parait un peu maladroit, peut être écrire "une fonction-composant", ou quelque chose qui s'y approche. L'idée c'est que c'est qu'on a une fonction qui décrit un composant.

Reply:
Ok, je vais le traduire par "une fonction composant" pour être en adéquation avec les traductions existantes comme par exemple : https://fr.reactjs.org/docs/hooks-faq.html#can-i-make-a-ref-to-a-function-component